core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-list/components/dot-template-selector/dot-template-selector.component.html

```diff
@@ -1,7 +1,8 @@
-<div class="wrapper">
+<div class="wrapper" (keydown.enter)="onClick()">
```

Comment:
You should not be listening to the enter key but instead focus the button, that's for sematic html and specially a11y.

More here:

https://user-images.githubusercontent.com/751424/185980076-997fe265-c7d1-4de2-aa95-d69dd60ef388.mp4

Also, we should think a way to do this globally, somehow.